src/components/grantsPage/GrantsPage.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
```

Comment:
These hooks are a serious game changer. Definitely makes accessing state THAT much easier.